server/stream.go

```diff
@@ -2389,7 +2395,7 @@ func (mset *stream) scheduleSetupMirrorConsumerRetryAsap() {
 	}
 	// To make *sure* that the next request will not fail, add a bit of buffer
 	// and some randomness.
-	next += time.Duration(rand.Intn(50)) + 10*time.Millisecond
+	next += time.Duration(rand.Intn(10*int(time.Millisecond))) + 10*time.Millisecond
```

Comment:
Put 10 inside, so int(10*time.Mliisecond)

Reply:
Done. I also updated the scheduleSourceConsumerRetryASAP to use the same backoff computation (it was still using the 10ms with 0 to 50uS jitter)